packages/m2m-gateway/src/model/errors.ts

```diff
@@ -89,3 +90,11 @@ export function clientAdminIdNotFound(
     title: "Client admin id not found",
   });
 }
+
+export function tenantQueryConflictError(): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: "IPACode and taxCode query parameters cannot be provided together",
+    code: "tenantQueryConflict",
+    title: "Tenant query conflict",
+  });
+}
```

Comment:
Proposing a more self-explaining name 

```suggestion
export function taxCodeAndIPACodeConflict(): ApiError<ErrorCodes> {
  return new ApiError({
    detail: "IPACode and taxCode query parameters cannot be provided together",
    code: "taxCodeAndIPACodeConflict",
    title: "Tax code and IPA code conflict in tenant query",
  });
}
```